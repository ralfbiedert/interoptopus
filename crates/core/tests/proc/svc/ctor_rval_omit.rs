use interoptopus::{ffi_service, ffi_type};

#[ffi_type]
enum Error {
    Something,
}

#[ffi_type(service)]
struct Service;

#[ffi_service]
impl Service {
    pub fn new() {}
}

#[allow(dead_code)]
fn main() {}
