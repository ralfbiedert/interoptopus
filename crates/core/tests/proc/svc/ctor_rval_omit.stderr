error: This method looks like a constructor, but it does not return ffi::Result<Self, _>
  --> tests/proc/svc/ctor_rval_omit.rs:13:12
   |
13 |     pub fn new() {}
   |            ^^^
